src/khoj/routers/helpers.py

```diff
@@ -825,10 +825,13 @@ async def send_message_to_model_wrapper(
     conversation_config: ChatModelOptions = await ConversationAdapters.aget_default_conversation_config(user)
     vision_available = conversation_config.vision_enabled
     if not vision_available and query_images:
+        logger.warning(f"Vision is not enabled for default model: {conversation_config.chat_model}.")
```

Comment:
IMO default vision model needs to also become an option for `ServerChatSettings`.

Reply:
Good idea given we have multiple vision model types supported now. But yeah, maybe better as a follow-up to allow setting the default vision model to use. Current behavior of using first added vision model is confusing